### PR TITLE
Generalize i18n strings in API Resource UIs

### PR DESCRIPTION
**Pull request by SujanSanjula96**
Related issues - https://github.com/wso2/product-is/issues/16344

### CODE REVIEW

**Comment by Achintha444** [comment]
Add changeset

File: apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-authorization.tsx [hunk 1]
```diff
@@ -94,7 +97,7 @@ export const AddAPIResourceAuthorization: FunctionComponent<AddAPIResourceAuthor
                 content={ 
                     (<>
                         { t("extensions:develop.apiResource.wizard.addApiResource.steps.authorization.form." + 
-                            "rbacMessage") }
+                            "rbacMessage", { productName: productName }) }
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-authorization.tsx:100]: When the key and variable name are same, we can add the variable only inside curly braces and js will automatically covert it to a key value pair.

```suggestion
                            "rbacMessage", { productName }) }
```

Let's update the other places as well :)

File: apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-basic.tsx [hunk 2]
```diff
@@ -146,7 +149,8 @@ export const AddAPIResourceBasic: FunctionComponent<AddAPIResourceBasicInterface
                         <Hint className="mb-0">
                             <Trans 
                                 i18nKey= { "extensions:develop.apiResource.wizard.addApiResource.steps.basic." +
-                                    "form.fields.identifier.hint" }>
+                                    "form.fields.identifier.hint" }
+                                tOptions={ { productName: productName } }>
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-basic.tsx:153]: In L155, there is a reference to the cloud offering. Shall we update it to "WSO2 Identity Server"?

File: apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-authorization.tsx (hunk 1 above)
Comment (SujanSanjula96) [reply apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-authorization.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/4369/commits/5f9e86491ef0a3af12f7235bb143cadc4347c5b8

File: apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-basic.tsx (hunk 2 above)
Comment (SujanSanjula96) [reply apps/console/src/extensions/components/api-resources/components/wizard/add-api-resource-steps/add-api-resource-basic.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/4369/commits/5f9e86491ef0a3af12f7235bb143cadc4347c5b8

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**